src/dbwrapper.cpp

```diff
@@ -147,6 +147,7 @@ static leveldb::Options GetOptions(size_t nCacheSize)
         // on corruption in later versions.
         options.paranoid_checks = true;
     }
+    options.max_file_size = 128 << 20;
```

Comment:
Should we make this a constant? Would it be appropriate to reuse `MAX_BLOCKFILE_SIZE`?

Reply:
+1 on a constant, but i don't think it's approprioate to reuse `MAX_BLOCKFILE_SIZE`, better to define a new one

Reply:
Added a relevant constant.